tutorials/beginners-cookbook/index.md

```diff
@@ -678,7 +783,7 @@ Export.table.toDrive({
 print(coll);
 ```
 
-## Bonus: Timelapse example
+## Bonus: Timelapse example [[Open In Code Editor](https://code.earthengine.google.com/fe801a6c86891d4e5eee260b85b0e716)]
```

Comment:
Could you turn this into an app instead of including a Code Editor link?

Reply:
@gino-m I wanted to keep a way to show the code since it is supposed to teach people how to do it themselves. Or I could just link to justin's tutorial (https://developers.google.com/earth-engine/tutorials/community/modis-ndvi-time-series-animation)? What do you think?

Reply:
Removed the link to the code editor.

Reply:
Do not include any Code Editor links, please.  We will not review them.